controllers/util/util.go

```diff
@@ -22,8 +22,9 @@ var (
 	ScannerMemRequest = flag.String("scanner-mem-request", "500Mi", "minimum memory request for scanner pods spawned by the eraser manager")
 	ScannerMemLimit   = flag.String("scanner-mem-limit", "2Gi", "limit on memory usage for scanner pods spawned by the eraser manager")
 
-	EraserImage = flag.String("eraser-image", "ghcr.io/azure/eraser:latest", "eraser image")
-	EraserArgs  = utils.MultiFlag([]string{})
+	EraserImage  = flag.String("eraser-image", "ghcr.io/azure/eraser:latest", "eraser image")
+	EraserArgs   = utils.MultiFlag([]string{})
+	OtlpEndpoint = flag.String("otlp-endpoint", "", "otel exporter otlp endpoint")
```

Comment:
any concerns on setting default to `otel-collector:4318`?

Reply:
Do we want metrics enabled by default? The collector is not part of the standard deployment or the helm chart so we need to be sure that the system can run with no collector

Reply:
agreed. can we add documentation on how to add it but we don't need to bundle it with eraser, would that work?

Reply:
that should work!